src/i18n.ts

```diff
@@ -131,3 +131,16 @@ export const t = (
   }
   return translation;
 };
+
+export const te = (
```

Comment:
I agree. Let's put the JSX logic into the `t()` function and not create a new one. We just need to make sure we don't return array of JSX elements unnecessarily (for performance reasons, I think).

I'd keep `t()` naming for now because it's short, but we can decide later.

Reply:
Yes, that would be optimal. I tried that strategy initially, but failed to satisfy the type-checker when `t()` is used as a component prop. Help is needed for a rewrite or a new PR.

Reply:
Ok, I'll take a look when free.

Reply:
resolved by f2e58cf